indico/modules/events/editing/client/js/management/editable_type/EditableList.module.scss

```diff
@@ -0,0 +1,84 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@import 'base/palette';
+
+$border: 1px solid $pastel-gray;
+
+.editable-topbar {
+    display: flex;
+    width: 1000px;
+    justify-content: space-between;
+}
+
+.editable-list {
+    .table {
+        font-family: 'Muli', 'Helvetica Neue', Arial, Helvetica, sans-serif;
+        margin-top: 20px;
+        margin-bottom: 50px;
+    }
+
+    .rowcolumn-tooltip {
```

Comment:
You can add `display: flex` here for both items (in one case the dot and the status and in another case the user icon and the user name) are aligned in a row.